scripts/blocks-utils.js

```diff
@@ -279,6 +281,47 @@ function decorateQuickLinks(main) {
   main.querySelectorAll('div.section-container > div > div').forEach(addQuickLinksMetadataForBlocks);
 }
 
+function readUpdateBlockConfig(block) {
+  const config = {};
+  block.querySelectorAll(':scope > div').forEach((row) => {
+    if (row.children) {
+      const cols = [...row.children];
+      if (cols[1]) {
+        const col = cols[1];
+        const name = toClassName(cols[0].textContent);
+        if (block.classList.contains('block') && block.hasAttribute('data-block-name')) {
+          cols[0].classList.add(`block-${block.getAttribute('data-block-name')}-${name}`);
```

Comment:
@sneharora1 @JiangLong2019 @sdmcraft 
this is what I was suggesting. Instead of ReadBlockConfig, we can use this method.
And we can identify our desired div and based on that we can get the value as well.